### PR TITLE
Fix for monta.ir

### PR DESCRIPTION
**Pull request by MiliAxe**
- Fix white images
- Fix images being black (almost transparent) in problems/choices/notes.
- Fix the problem of the answer sheet being white by default
- Fix graphs being white

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -6518,6 +6518,42 @@ CSS
...
+.GO5O4PB-gc-i
+.GO5O4PB-gc-e
+.GO5O4PB-gc-i > div
+.GO5O4PB-gc-Eb > div > a
+img.is-rounded
+img.is-128x128
+
+CSS
+.GO5O4PB-gc-n {
+ background-image: none; !important
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6552]:
```suggestion
    background-image: none !important;
```

**Review by Gusted — CHANGES_REQUESTED** [review]
See suggestion.

**Comment by MiliAxe** [comment]
Thanks for the suggestion.
The problem has been resolved.

**Review by Gusted — APPROVED** [review]
Thank you!

**Pull request merged**